tests/io/v0_0_0/test_missing_shape.py

```diff
@@ -17,16 +18,16 @@
 class TestMissingShape(unittest.TestCase):
     def test_tileset_without_shapes(self):
         image = slicedimage.TileSet(
-            ["x", "y", "ch", "hyb"],
+            [DimensionNames.X, DimensionNames.Y, "ch", "hyb"],
```

Comment:
Is there not also DimensionNames. ROUND, DimensionNames.CH ?

Reply:
Nope, because those are not dimensions that are in the spec for slicedimage.

Reply:
ohhh got it ok